txprocessor/txprocessor.go

```diff
@@ -1245,7 +1254,11 @@ func (tp *TxProcessor) applyExit(coordIdxsMap map[common.TokenID]common.Idx,
 			}
 
 			// accumulate the fee for the Coord account
-			accumulated := tp.AccumulatedFees[accCoord.Idx]
+			accumulated, ok := tp.AccumulatedFees[accCoord.Idx]
+			if !ok {
+				accumulated = big.NewInt(0)
+				tp.AccumulatedFees[accCoord.Idx] = accumulated
+			}
 			accumulated.Add(accumulated, fee)
```

Comment:
What you think refactor this snippet into a method from `tp` or outside it?

Reply:
Nice point, I agree!!! But I will not refactor this stuffs into the TxProcessor now. There are too many things to do inside it, I only added this to avoid panic because this check was inside the TxSelector, but it's related to the TxProcessor.